src/ecrc-frontend/src/components/base/loader/Loader.stories.js

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+
+import Loader from "./Loader";
+
+export default {
+  title: "Loader",
+  component: Loader
+};
+
+export const Default = () => <Loader />;
```

Comment:
do we need a mobile story here?

Reply:
The component is so small (40px) that I dont think it would break on a mobile screen. I can add it if its needed though.